pkg/operator/ceph/csi/spec.go

```diff
@@ -243,6 +247,20 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		deployProvSTS = true
 	}
 
+	tp.ResizerImage = os.Getenv("ROOK_CSI_RESIZER_IMAGE")
+	if tp.ResizerImage == "" {
+		tp.ResizerImage = defaultResizerImage
+	}
+
+	if ver.Major == KubeMinMajor {
```

Comment:
This should not be `==` so that k8s `0.X` and `2.X` would be supported if necessary.

Reply:
this is for logging if the kube version is less than 1.15 or 1.16 we are logging that expansion is not supported

Reply:
Right. But for k8s versions `0.X`, this feature is also not supported. We can't ignore the major number. The correct checks are 
```go
if ver.Major < KubeMinMajor || ver.Major == KubeMinMajor && ver.Minor < kubeMinVerForXXXXRestore { 
    // unsupported
}
```

Reply:
@BlaineEXE updated PTAL :)